src/__internal__/form-field/form-field.spec.tsx

```diff
@@ -0,0 +1,253 @@
+import React from "react";
+import { mount } from "enzyme";
+import CarbonProvider from "../../components/carbon-provider";
+import FormField, { FormFieldProps } from "./form-field.component";
+import FieldHelp from "../field-help";
+import { FieldLineStyle } from "./form-field.style";
+import Label from "../label";
+import { TabContext } from "../../components/tabs/tab";
+import {
+  assertStyleMatch,
+  mockMatchMedia,
+} from "../../__spec_helper__/test-utils";
+
+const setError = jest.fn();
+const setWarning = jest.fn();
+const setInfo = jest.fn();
+
+interface MockComponentProps {
+  children?: React.ReactNode;
+}
+const MockComponent = ({ children }: MockComponentProps) => {
```

Comment:
maybe I've missed something but I don't understand the purpose of introducing `MockComponent` to wrap the content in a plain `div`. I've tried removing it and all tests still pass and with 100% coverage - so I'd rather not have it to avoid bloating the code for no reason.

Reply:
This was mainly so the our use of `FormField` in `render()` matches how `FormField` would be used in practice, by wrapping `FormField` in some component and setting its `data-component` prop to the name of that component. However, I agree looking back at this the code does look bloated - so will modify this 👍🏼